include/ebpf_program_types.h

```diff
@@ -81,7 +81,7 @@ typedef union _program_data_capabilities
     uint32_t value;
     struct
     {
-        bool supports_context_header : 1; // Program supports context header.
+        bool reserved : 1; // Reserved (must be zero).
```

Comment:
For my understanding - why must this be 0?

Reply:
After looking through the code, it seems like this is to help validation that this field is stricly set to 0, as opposed to a previous version which may have used it, is that correct? 

Do we even need this? Can this just be part of the breaking change that removes this field entirely?

Reply:
If we need program_data_capabilities_t to be of fixed size as new capabilities are needed over time, then this is fine as is.